docs/source/transactions.md

```diff
@@ -0,0 +1,49 @@
+---
+html_meta:
+  "description": "The @transactions endpoint exposes transactions those have been made through the plone website."
+  "property=og:description": "The @transactions endpoint exposes transactions those have been made through the plone website."
+  "property=og:title": "Transactions"
+  "keywords": "Plone, plone.restapi, REST, API, Transactions"
+---
+
+# Transactions
+
+The `@transactions` endpoint exposes transactions those have been made through the plone website.
+Each change thorough the plone website is listed.
+It also allows to revert transactions so that previous version of the website can be achievable.
```

Comment:
Should be: "It also allows to revert transactions so that the Plone website can be reverted to a previous state."